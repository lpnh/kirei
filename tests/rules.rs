mod helper;

// === Whitespace

#[test]
fn html_trailing_space() {
    format_and_snapshot!("rules/whitespace/html_trailing_space.html");
}

#[test]
fn html_blank_line() {
    format_and_snapshot!("rules/whitespace/html_blank_line.html");
}

#[test]
fn html_entity() {
    format_and_snapshot!("rules/whitespace/html_entity.html");
}

#[test]
fn tmpl_trailing_space() {
    format_and_snapshot!("rules/whitespace/tmpl_trailing_space.html");
}

#[test]
fn tmpl_empty_comment() {
    format_and_snapshot!("rules/whitespace/tmpl_empty_comment.html");
}

// === Delimiter Padding

#[test]
fn no_padding_expression() {
    format_and_snapshot!("rules/delimiter_padding/no_padding_expression.html");
}
