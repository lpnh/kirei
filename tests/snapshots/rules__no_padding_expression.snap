---
source: tests/rules.rs
description: "<div>{{user.name}}</div>\n"
---
<div>{{ user.name }}</div>
